### ISSUE TITLE
coneMosaic.spatialDensity vs coneMosaic.spatialDensity_

### ISSUE THREAD

**Issue by DavidBrainard** [isetbio/isetbio#206]
I don't understand the rational for both coneMosaic.spatialDensity and coneMosaic.spatialDensity_.  Probably there is a good one, it just isn't immediately clear to me.  I would like to understand it so we can add comments that explain it better, and also so we can think hard about ways that the two could get out of whack.

Note that if the user writes a new pattern in after the mosaic object is created, I believe that currently both spatialDensity properties will not be accurate.  But I'm not completely sure.

**Comment by wandell** [comment]
Yes, there is one that HJ has.  I meant to comment.  Let's let @hjiang36 explain ... . Roughly - one refers to the internal high resolution computation not seen by outside functions and the other refers to what you can set externally and is available outside the function.  

I think this is common practice amongst people who program all the time ... it was new to me.
